main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -0,0 +1,670 @@
+		"column": "Colonne",
+		"include": "Inclure",
+		"as-property": "Comme propriété",
+		"contact-service": "Connexion au service de réconciliation",
+		"error-contact": "Erreur lors de la connexion au service de réconciliation",
+		"fb-recon": "Réconciliation basée sur des requêtes Freebase",
+		"recon-col": "Réconcilier la colonne",
+		"pick-service": "Choisir un service ou une extension à gauche",
+		"add-recon-srv": "Ajouter un service de réconciliation avec un espace de noms",
+		"namespace": "Espace de noms",
+		"ent-type": "Type des entités (facultatif)",
+		"add-std-srv": "Ajouter un service de réconciliation standard",
+		"enter-url": "Indiquer l’URL du service",
+		"specify-ns": "Merci d’indiquer un espace de nom.",
+		"cell-contains": "Chaque cellule contient :",
+		"fb-id": "un ID Freebase, par exemple /en/solar_system",
+		"fb-guid": "un GUID Freebase, par exemple #9202a8c04000641f80000000000354ae",
+		"fb-key": "une clé Freebase",
+		"fb-en-ns": "l’espace de noms Wikipedia English",
+		"this-ns": "cet espace de noms :",
+		"max-candidates" : "Nombre maximal de candidats renvoyés",
+		"service-title": "Services"
+	},
+	"core-util-enc": {
+		"select-enc": "Choisir l’encodage",
+		"common": "Encodage courants",
+		"all": "Tous les encodages",
+		"encoding": "Encodage",
+		"aliases": "Alias",
+		"today": "aujourd’hui",
+		"yesterday": "hier",
+		"days-ago": "jours avant",
+		"week-ago": "semaine dernière",
+		"weeks-ago": "semaines avant",
+		"month-ago": "mois dernier",
+		"months-ago": "mois avant",
+		"year-ago": "année dernière",
+		"years-ago": "années avant",
+		"working": "En fonctionement",
+		"invalid-date": "Chaîne de date incorrecte"
+	},
+	"core-views": {
+		"edit-cell": "Éditer cette cellule",
+		"choose-match": "Choisir une nouvelle correspondance",
+		"match-all-cells": "Apparier ce sujet et cette cellule et toutes celles identiques",
+		"match-this-cell": "Apparier ce sujet et cette cellule",
+		"create-topic-cells": "Créer un nouveau sujet pour cette cellule et toutes celles identiques",
+		"create-topic-cell": "Créer un nouveau sujet pour cette cellule",
+		"create-topic": "Créer un nouveau sujet",
+		"search-match": "Chercher les correspondances",
+		"not-valid-number": "Nombre invalide.",
+		"not-valid-date": "Date invalide.",
+		"match-this": "Apparier seulement cette cellule",
+		"match-other": "Apparier les autres cellules avec le même contenu",
+		"search-for": "Chercher",
+		"match-cell": "Apparier cette cellule",
+		"match-identical": "Apparier toutes les cellules identiques",
+		"matched": "appariée",
+		"new": "nouveau",
+		"to-be-recon": "à réconcilier",
+		"facet": "Facette",
+		"edit-cells": "Éditer les cellules",
+		"edit-column": "Éditer la colonne",
+		"transpose": "Transposer",
+		"sort": "Trier",
+		"collapse-expand": "Plier/déplier les colonnes pour rendre l’affichage des données plus pratique",
```

Comment:
This PR was merged 4 days ago.  Please open a new issue (or better yet, pull request).